devel/ghidra/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           app 1.0
+
+name                ghidra
+homepage            https://ghidra-sre.org/
+master_sites        ${homepage}
+version             9.2.2
+use_zip             yes
+distfiles           ${name}_${version}_PUBLIC_20201229.zip
+checksums           rmd160  2b7a239329ba515010ff52725da08eb656f168f7 \
+                    sha256  8cf8806dd5b8b7c7826f04fad8b86fc7e07ea380eae497f3035f8c974de72cf8 \
+                    size    317805407
+revision            0
+
+categories          devel
+maintainers         {1e0.co.uk:dev @hexagonal-sun} openmaintainer
+description         A software reverse engineering (SRE) suite of tools developed by NSA's \
+                    Research Directorate in support of the Cybersecurity mission
+long_description    ${description}
+license             Apache
+platforms           darwin
+
+depends_lib-append  port:zulu-jdk11
+use_configure       no
+
+build {}
```

Comment:
So this is a pre-built application? Can it be compiled from source?

Reply:
What happens if the universal variant is chosen?

Reply:
I've been trying all day to get GHIDRA to compile from source but I'm really strulggling.  I believe the build has been simplified for 9.3.  For now building from the binaries seems to work fine.

I'm not sure what happens if a universal variant is chosen.

Reply:
So make sure that the universal variant is not available. It is normally created by default.